src/index.html

```diff
@@ -8,13 +8,15 @@
 <body>
   <header>
     <nav>
-      <li>
-        <img id="logo" src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/98/International_Pok%C3%A9mon_logo.svg/2560px-International_Pok%C3%A9mon_logo.svg.png" alt="Pokemon">
-      </li>
+      <div id="logo"></div>
       <ul>
-        <li>Pokemon</li>
+        <li class="nav-list">Pokemon</li>
```

Comment:
Thank you for fixing that, I didn't notice I had improperly nested the `li` till I had submitted the pull request.